experiments/experiments-impl/src/main/java/com/duckduckgo/experiments/impl/VariantManagerImpl.kt

```diff
@@ -46,7 +46,7 @@ class VariantManagerImpl @Inject constructor(
         val activeVariants = convertEntitiesToVariants(experimentVariantRepository.getActiveVariants())
         val currentVariantKey = store.variant
 
-        if (currentVariantKey == DEFAULT_VARIANT.key) {
+        if (!experimentVariantRepository.isVariantManagerConfigReady() || currentVariantKey == DEFAULT_VARIANT.key) {
```

Comment:
Ditto, I don't think we need the `isVariantManagerConfigReady` check at all

Reply:
Outdated